validphys2/src/validphys/covmats.py

```diff
@@ -190,7 +191,6 @@ def dataset_inputs_covmat_from_systematics(
     special_corrs = []
     block_diags = []
     weights = []
-
```

Comment:
I believe we need them because we want to separate the case in which only the additive part of the uncertainties are retained in the covmat and the case in which the covmat is complete. But maybe I misunderstood the question.

Reply:
Being the question attached to a deleted blank line, my naive interpretation would be: "Why are you deleting the blank line?"

Reply:
ok easy ty 😂